src/ctapipe/image/cleaning.py

```diff
@@ -112,6 +113,41 @@ def tailcuts_clean(
         )
 
 
+def bright_cleaning(image, threshold, fraction):
+    """
+    Clean an image by removing pixels below a fraction of the mean charge
+    in the 3 brightest pixels.
+
+    Select no pixels instead if the mean charge of the brightest pixels
+    are below a certain threshold.
+
+    Parameters
+    ----------
+    image: array
+        pixel values
+    threshold: `float`
+        Minimum average charge in the 3 brightest pixels to apply
+        cleaning
+    fraction: `float`
+        Pixels below fraction * (average charge in the 3 brightest pixels)
+        will be removed from the cleaned image
+
+    Returns
+    -------
+    A boolean mask of *clean* pixels.
+
+    """
+    mean_3_max_signal = np.mean(n_largest(3, image))
+
+    if mean_3_max_signal < threshold:
+        return np.zeros(image.shape, dtype=bool)
```

Comment:
Shouldn't this return `np.ones` ?  I.e. if we the bright pixels are not bright enough, we don't remove any pixels?

Reply:
yes you are right